frontend/routes/components/selectField/selectField.component.tsx

```diff
@@ -108,7 +108,7 @@ export class SelectField extends React.PureComponent<any, any> {
 		};
 
 		return (
-			<Select className={className} {...selectProps} MenuProps={customMenuProps}>
+			<Select classes={classes} className={className} {...selectProps} MenuProps={customMenuProps}>
```

Comment:
I think you no need extra `classes` prop. It's included on `selectProps`, which is also passed to this component ;)